concepticondata/conceptlists/Vergallito-2020-1121.tsv

```diff
@@ -0,0 +1,1122 @@
+Vergallito-2020-1121-960	960	speranza	hope	1092	HOPE	8	n	0.9666666667	572.0344828	1	548.1428571	-0.4075896434	-0.1140823301	2.58	0.21	0.3	0.18	2.26	a	43.39963834	2.58	0.18	1.106	3.454055587
+Vergallito-2020-1121-961	961	speranzoso	hopeful			10	a	0.9666666667	723.9310345	1	593.6551724	0.5018174307	0.2214164096	3.18	0.11	0.34	0.18	2.96	a	45.34711965	3.18	0.11	1.354	4.362808728
+Vergallito-2020-1121-962	962	spiaggia	beach	3460	BEACH	8	n	0.9333333333	561.25	1	565.3793103	-0.447015002	0.1629877255	3.42	0.32	4.02	2.47	4.82	v	29.90033223	4.82	0.32	3.01	7.569180933
+Vergallito-2020-1121-963	963	spillatrice	stapler			11	n	0.9333333333	846.0714286	1	692.5555556	1.090986962	0.9817219293	2.75	0.11	4.02	0.14	4.66	v	38.95547945	4.66	0.11	2.336	6.743159497
+Vergallito-2020-1121-964	964	spina	thorn	124	THORN	5	n	0.9333333333	604.1428571	1	560.826087	-0.3158959661	0.07879119659	0.44	0.26	4.7	0.07	4.6	h	45.97815293	4.7	0.07	2.014	6.596673404
+Vergallito-2020-1121-965	965	spirito	spirit	53	SPIRIT	7	n	1	604.6666667	1	616.8	-0.1649560184	0.6958787809	1.04	0.09	0.07	0.3	0.81	a	41.99134199	1.04	0.07	0.462	1.356723996
+Vergallito-2020-1121-966	966	sporcizia	filth			9	n	0.9	715.5555556	1	628.3214286	0.4723929191	0.5132527611	0.44	0.47	2.79	3.56	4.61	v	35.1305813	4.61	0.44	2.374	6.490323567
+Vergallito-2020-1121-967	967	sporco	dirt	25	DIRT	6	a	1	573.2	1	527.7307692	-0.337867294	0.001955350988	0.47	0.84	2.65	3.35	4.74	v	35.43568465	4.74	0.47	2.41	6.452836586
+Vergallito-2020-1121-968	968	sposa	bride	1266	BRIDE	5	n	1	522.0333333	1	530.6923077	-0.8097101641	-0.265580936	2.19	0.21	2.19	0.74	4.81	v	45.36489152	4.81	0.21	2.028	5.772347876
+Vergallito-2020-1121-969	969	sposo	spouse	2514	SPOUSE	5	n	1	596.4666667	1	518.2758621	-0.2783315877	-0.1950769911	2.58	0.19	1.77	0.72	4.58	v	44.61382114	4.58	0.19	1.968	5.5964453
+Vergallito-2020-1121-970	970	spray	spray			5	n	0.9333333333	609.7857143	1	544.7777778	-0.06087916417	-0.1632072705	3.53	0.68	2.82	3.47	3.96	v	22.68326418	3.96	0.68	2.892	6.971240923
+Vergallito-2020-1121-971	971	sprezzante	scornful			10	a	0.9333333333	793.5714286	1	584.6428571	1.037066756	0.2629499711	3.55	0.45	0.73	0.38	3.53	a	36.68981481	3.55	0.38	1.728	5.09344677
+Vergallito-2020-1121-972	972	squalo	shark	1110	SHARK	6	n	1	592.3	1	529.3461538	-0.242439717	-0.08221379666	1.53	0.74	2.63	0.56	4.75	v	41.03819785	4.75	0.56	2.042	5.716773566
+Vergallito-2020-1121-973	973	stagnante	stagnant			9	a	0.7	867.9047619	1	661.8571429	1.136996199	0.6961938073	0.73	0.41	0.95	3.32	4.13	v	38.99371069	4.13	0.41	1.908	5.448192361
+Vergallito-2020-1121-974	974	stampella	crutch			9	n	0.9	679.4444444	1	553.7407407	0.3462687248	0.03663315702	1.56	0.05	4.25	0.16	4.89	v	44.36296975	4.89	0.05	2.182	6.666055805
+Vergallito-2020-1121-975	975	stampo	mold	289	MOLD	6	n	0.9666666667	580.6206897	1	535.0689655	-0.2763992547	-0.158060331	0.85	0.07	3.15	0.2	4.46	v	50.28636884	4.46	0.07	1.746	5.530054249
```

Comment:
This is "mold" as the noun, the container used to mold things, not the verb (as in Concepticon)